docker/run.sh

```diff
@@ -1,13 +1,6 @@
-#!/bin/sh
+#!/bin/sh -x
 
 # Set password
 echo "root:${PASSWORD}" | chpasswd
 
-if [ -z ${PORT} ]
-then
-  echo "Starting on default port: 57575"
-  /opt/app/butterfly.server.py --unsecure --host=0.0.0.0
-else
-  echo "Starting on port: ${PORT}"
-  /opt/app/butterfly.server.py --unsecure --host=0.0.0.0 --port=${PORT}
-fi
\ No newline at end of file
+/opt/app/butterfly.server.py $@
```

Comment:
it may better to do an `exec` here; doing so makes `python` the primary process of the container (instead of `/bin/sh`).

Without this;

```
ps aux

root         1  0.1  0.0   4448   764 ?        Ss   09:43   0:00 /bin/sh -x /opt/run.sh --unsecure --host=0.0.0.0 --port=57575 --login
root         9  1.0  0.4  74504 18820 ?        S    09:43   0:00 python /opt/app/butterfly.server.py --unsecure --host=0.0.0.0 --port=57575 --login
```

after adding `exec`:

```
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root         1  1.6  0.4  73860 18088 ?        Ss   09:46   0:00 python /opt/app/butterfly.server.py --unsecure --host=0.0.0.0 --login
```